adapters/risc0/src/host.rs

```diff
@@ -9,12 +9,14 @@ use sov_rollup_interface::zk::{Zkvm, ZkvmHost};
 
 use crate::Risc0MethodId;
 
+/// Allows to run prover code and send data to ZkVM
```

Comment:
nit: This comment is a little terse. How about...

"The 'host' of a Risc0 zkVM. Generates a SNARK by running the target code on a virtual RISC-V CPU. The host is responsible for sending the "witness" data to the zkVM guest."